source/dom.js

```diff
@@ -63,8 +63,8 @@ export const getJSON = (url, callback) => {
     const name = encodeURIComponent(`random_fun_${++gid}`);
 
     const concreteUrl = url.replace(
-        /callback=(\?)/,
-        `callback=__likelyCallbacks.${name}`
+    /(callback|jsonp)=(\?)/,
+    `$1=__likelyCallbacks.${name}`
```

Comment:
Did you drop these tabs on purpose?

Reply:
Outdated. I've been looking at each commit separately so didn't spot the last change.